src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTestTransformer.java

```diff
@@ -16,11 +16,13 @@
 
 import com.google.api.codegen.InterfaceView;
```

Comment:
The diff for this class is difficult to walk through because I reorganized it to be much more clear. There are improvements throughout, but the most important are in the new `createSmokeTest.*` methods.